packages/performance/src/resources/trace.test.ts

```diff
@@ -117,6 +117,13 @@ describe('Firebase Performance > trace', () => {
       expect(trace.getMetric('cacheHits')).to.eql(600);
     });
 
+    it('increments metric value as an integer even if the value is provided in float.', () => {
+      trace.incrementMetric('cacheHits', 200);
```

Comment:
There are two scenarios I think worth checking:

1. Can the value be negative? If so how does the flooring look like?
2. What if the integer excess number's range limit?

Reply:
I don't think that would be the scope of this PR. For both of the above, I would prefer to retain the current functionality. Let me know what you think.

Reply:
Since we added `Math.floor()` logic, it comes to my question that how will the negative value be changed if we provide negative value, unless we don't allow negative value to be provided.

Reply:
Added a test for negative values provided as inputs.